dask/dataframe/tests/test_dataframe.py

```diff
@@ -21,6 +21,7 @@
 from dask.dataframe import methods
 from dask.dataframe.utils import (assert_eq, make_meta, assert_max_deps,
                                   PANDAS_VERSION)
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
I don't think we want this. Rather, we should

1. assert that the warning is emitted if we're the ones emitting the warning
2. ensure that the code in `dask/dataframe` doesn't case a warning.